.github/workflows/ci.yml

```diff
@@ -252,6 +264,13 @@ jobs:
             ctest . -j2 --output-on-failure
           fi
 
+      - name: Run lcov
+        if: matrix.build == 'coverage'
+        run: |
+          lcov --directory ./app-build/ --capture --output-file ./app-build/app_coverage --gcov-tool gcov-10
+          lcov --remove ./app-build/app_coverage '/usr/*' '${{ github.workspace }}/app/lib/*' '${{ github.workspace }}/app-build/vendor/*' --output-file ./app-build/app_coverage
+          lcov --list ./app-build/app_coverage
```

Comment:
seqan3 we integrated that into cmake in "test/coverage", but I guess it is fine for now to just have this in CI

Reply:
Yes, but it doesn't seem trivial to reuse it. As far as I can see it relies on a certain way the tests are configured...